kubernetes-addons/Observability/chatqna/tgi-gaudi-serviceMonitor.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app.kubernetes.io/name: tgi-gaudi-exporter
+    app.kubernetes.io/version: v0.0.1
+    release: prometheus-stack
+  name: tgi-gaudi-exporter
+  namespace: monitoring
+spec:
+  namespaceSelector:
+      any: true
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: tgi
```

Comment:
You need to use all selectorLabels in case there are multiple instances in same cluster, see:
https://github.com/opea-project/GenAIInfra/blob/main/helm-charts/common/tgi/templates/servicemonitor.yaml

(Same goes for rest of the monitors.)

Reply:
Please all all the selector labels corresponding to your chatqna deployment method described in the README, otherwise you could end up monitor tgi pod for other workload

Reply:
already added the "app.kubernetes.io/instance: chatqna" to each serviceMonitor